### PR TITLE
go-import need to be higher up in the <head> to work correctly

### PR DESCRIPTION
**Pull request by brimstone**
I believe this fixes the `go get` problem.

### CODE REVIEW

File: layouts/404.html [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <!DOCTYPE html>
 <html lang="en">
     <head>
-      {{partial "head" .}}
```
Comment (fuskovic) [inline layouts/404.html:4]: I think the right thing to do is put the meta-tag in the partial itself which is located at /layouts/partials/head.html

**Review by VinozzZ — APPROVED** [review]

**Comment by VinozzZ** [comment]
Can you resolve the conflicts?

**Review by egonelbre — APPROVED** [review]

**Review by fuskovic — COMMENTED** [review]
This fix has already been applied to production
https://storjlabs.atlassian.net/browse/MAR-1584

Is it not working ?

**Comment by fuskovic** [comment]
I confirmed with @jtolds https://storjlabs.atlassian.net/browse/MAR-1584 did solve this issue and I simply forgot to close this PR and I did tell @brimstone in a one off that I would just lump this in my next commit since I was about to do that soon at the time anyway. Sorry for the confusion. Closing this PR.

**Pull request closed without merging**